### PR TITLE
Fixes dev for windows, adds license

### PR DESCRIPTION
**Pull request by wgester**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "browserify src/index.js -g uglifyify | uglifyjs --screw-ie8 -m -c dead_code,sequences,conditionals,booleans,unused,if_return,join_vars,drop_debugger > public/bundle.js",
     "watch": "watchify src/index.js -d -v -o public/bundle.js",
     "start": "npm run build && serve --compress public/ -p 1618",
-    "dev": "npm run watch & serve public/ -p 1618",
+    "dev": "node bin/dev.js",
```
Comment (TheAlphaNerd) [inline package.json]: if we make the script executable we can so ./bin/dev

**Comment by TheAlphaNerd** [comment]
LGTM

**Comment by TheAlphaNerd** [comment]
:+1:

**Comment by wgester** [comment]
Done

**Comment by TheAlphaNerd** [comment]
landed as

87dca31d2858a5bd0d2066ffdbf793c44fbc5566
3dc5fc2de764d05e4f96586e5ab89f5e8a444f02
de032f248be20515b64ee71efb3db45953929075
445de375829e623f4bc19f5b8e1d9f01f3cda73f
9b7f1f2974ac8e4959ec51ffba145013697b76ee

**Pull request closed without merging**